api/src/main/java/org/openmrs/module/coreapps/htmlformentry/EncounterDiagnosesByObsElement.java

```diff
@@ -138,6 +144,39 @@ public String generateHtml(FormEntryContext context) {
         }
     }
 
+    /**
+     * This method recieves a comma seperated list of diagnosis sets which are verified and then
+     * returns as comma seperated list of their uuids. diagnosisSetIds may be concept's 1). uuid
+     * 2). mapping, 3). id
+     *
+     * @param diagnosisSetIds
+     * @return uuids
+     */
+    private String validateAndFormat(String diagnosisSetIds) {
```

Comment:
??

Reply:
This is here for the `EncounterDiagnosesByObsElement` as explained above.